package-lock.json

```diff
@@ -8226,20 +8226,20 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```

Comment:
This `package-lock.json` file doesn't need to change. The way to ensure we have the right package-lock changes for packages is to run `npm run install:packages` on the branch.